CONTRIBUTING.md

```diff
@@ -44,8 +44,13 @@ The output artifacts will be in `instrumentation/build/outputs/`.
 We use [spotless](https://github.com/diffplug/spotless) to enforce a consistent code style
 throughout the project. This includes reformatting (linting) of both source code and markdown.
 
-Before submitting a PR, you should ensure that your code is linted by
-running `./gradlew spotlessApply` and committing your changes.
+Before submitting a PR, you should ensure that your code is linted. We use the
+[spotless gradle plugin](https://github.com/diffplug/spotless/tree/main/plugin-gradle) to
+make this easy. You should run it like this:
+
+```
+./gradlew spotlessApply
+```
```

Comment:
What I like doing is having a `Makefile` for such things and git pre-commit hooks, so it's "forced" to fix such things before committing.
Even better if CI runs auto-fix andcommitst changes ^^
Not a blocker, just ideas.

Reply:
Yeah, or something like this: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/.githooks/pre-commit